content/departments/product-engineering/engineering/cloud/devops/update_sg_website_config.md

```diff
@@ -8,18 +8,15 @@ Site configuration for sourcegraph.com is split into two files. One contains non
 
 ### Non-sensitive configurations
 
-Non-sensitve configurations are stored in a [ConfigMap](https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/release/base/frontend/non-sensitive-site-config.ConfigMap.yaml).
+Non-sensitive configurations and env vars are stored in an [overlay](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/deploy-sourcegraph-cloud%24+file:%5Eoverlays/prod/frontend&patternType=literal).
```

Comment:
I think we should point people directly to the file, no everyone understand how kustomize work

https://github.com/sourcegraph/deploy-sourcegraph-cloud/blob/filip/prod_overlay_frontend/overlays/prod/frontend/files/site.json